README.md

```diff
@@ -155,6 +155,16 @@ This package comes with three `CrawlProfiles` out of the box:
 - `CrawlInternalUrls`: this profile will only crawl the internal urls on the pages of a host.
 - `CrawlSubdomainUrls`: this profile will only crawl the internal urls and its subdomains on the pages of a host.
 
+### Ignoring robots.txt and robots meta
+
+By default, the crawler will respect robots data. It is possible to disable these checks like so:
```

Comment:
Add some extra info here on what a robots.txt and header is? Also mention that the actual check is being done by our robots-txt package.

Reply:
Also done.